installing/installing_ibm_cloud_public/configuring-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+[id="configuring-iam-ibm-cloud"]
+= Configuring IAM for IBM Cloud
+include::modules/common-attributes.adoc[]
+:context: configuring-iam-ibm-cloud
+
+toc::[]
+
+In environments where the cloud identity and access management (IAM) APIs are not reachable, you must put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
+
+include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
+
+.Additional resources
+
+For a detailed description of all available CCO credential modes and their supported platforms, see xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[About the Cloud Credential Operator].
+
+include::modules/cco-ccoctl-configuring.adoc[leveloffset=+1]
+include::modules/refreshing-service-ids-ibm-cloud.adoc[leveloffset=+1]
+//include::modules/manually-maintained-credentials-upgrade.adoc[leveloffset=+1]
+// Will need to revisit upgrade scenario for IBM Cloud; not needed until OCP 4.11. Tentative instructions have been added for reference later.
+
+[id="next-steps_configuring-iam-ibm-cloud"]
+== Next steps
```

Comment:
[id="next-steps_configuring-iam-ibm-cloud"]
== Next steps

per updated guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
@ahardin-rh not sure if I need an update here. The heading already has an anchor ID that adheres to the standards.